example/src/screens/Examples/AwareScrollView/index.tsx

```diff
@@ -33,6 +36,7 @@ export default function AwareScrollView({ navigation }: Props) {
     <KeyboardAwareScrollView
       testID="aware_scroll_view_container"
       bottomOffset={50}
+      enableKeyboardHandling={isFocused}
```

Comment:
Maybe let's simply use `enabled` property?

Reply:
we already have `enabled` prop in ScrollView, I think it's better to not override it